src/SentryOne.UnitTestGenerator.Specs/MethodBasedStrategySteps.cs

```diff
@@ -106,6 +106,11 @@ public void WhenIGenerateTestsForTheMethod(string strategy)
                 generationStrategy = new StringParameterCheckMethodGenerationStrategy(frameworkSet);
             }
 
+            if (strategy == "MappingMethodGenerationStrategy")
```

Comment:
Should we use nameof(MappingMethodGenerationStrategy) here instead?

Reply:
Looking at it, there's a bunch of improvements we could make to the Specs - so rather than make one small improvement I think we should take a pass on it when we have a decent amount of time to do so...